src/main/java/courseManager/Outline.java

```diff
@@ -0,0 +1,98 @@
+package courseManager;
+
+import weightScheme.WeightScheme;
+
+import java.util.ArrayList;
+
+public class Outline {
+    private double runningGrade;
+    private double hypothethicalGrade;
+
+    private double percentageCompleted;
+    private int numberOfAssessmentsCompleted; /*do these need to be their own instance variables?*/
+    private ArrayList<Assessment> assessments;
+
+    public Outline () {
+        this.runningGrade = 0.0f;
+        this.hypothethicalGrade = 0.0f;
+        this.assessments = new ArrayList<>();
+    }
+
+    public void setRunningGrade(double runningGrade) {
+
+        this.runningGrade = runningGrade;
+    }
+
+    public void setHypothethicalGrade(double hypothethicalGrade) {
+
+        this.hypothethicalGrade = hypothethicalGrade;
+    }
+
+    public void addAssessment(Assessment assessment) {
+
+        assessments.add(assessment);
+    }
+
+    public void removeAssessment(Assessment assessment) {
+
+        assessments.remove(assessment);
+    }
+
+    public void editAssessment(int index, String title) {
+        assessments.get(index).setTitle(title);
+    }
+    public void editAssessment(int index, WeightScheme scheme) {
+        assessments.get(index).setWeightScheme(scheme);
+    }
+    public void editAssessment(int index, String title, WeightScheme scheme) {
+        assessments.get(index).setTitle(title);
+        assessments.get(index).setWeightScheme(scheme);
+    }
+
+    public Assessment getAssessmentData(int index) {
+        return assessments.get(index);
```

Comment:
Similar to instanceList comment, possible index error

Reply:
done!